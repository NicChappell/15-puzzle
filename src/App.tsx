import React from "react";
import { GameBoard } from "./components";
import "./App.css";

const App = () => {
  return (
    <main id="app">
      <GameBoard />
    </main>
  );
};

export default App;
