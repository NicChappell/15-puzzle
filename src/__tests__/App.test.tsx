import React from "react";
import { render } from "@testing-library/react";
import App from "../App";

describe("App", () => {
  it("renders", () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
