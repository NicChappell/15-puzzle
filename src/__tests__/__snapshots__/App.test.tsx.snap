// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div>
  <main
    id="app"
  >
    <div
      class="game-board"
    >
      <p>
        Game Board
      </p>
    </div>
  </main>
</div>
`;
